frontend/src/app/libs/user-profile-page/service/user-converter.service.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { TestBed } from '@angular/core/testing';
+
+import { UserConverter } from './user-converter.service';
+import { UserDto } from '../model/user-dto';
+import { User } from '../../auth/models/user';
+import { UserChanges } from '../model/user-changes';
+
+describe('UserConverterService', () => {
+	let service: UserConverter;
+	let createdUser: User;
+	let createdUserChanges: UserChanges;
+
+	const userDto: UserDto = {
+		email: '1@1.com',
+		isAdmin: false,
+		_id: 'test',
+		firstName: 'undefined',
+		mobilePhone: 'undefined',
+		secondName: 'undefined',
+		password: 'undefined',
+	};
+
+	const user: User = {
+		email: '1@1.com',
+		isAdmin: false,
+		id: 'test',
+		firstName: 'undefined',
+		mobilePhone: 'undefined',
+		secondName: 'undefined',
+		password: 'undefined',
+	};
+
+	const userChanges: UserChanges = {
```

Comment:
seems like you can use just `User` interface here, add few necessary fields and all will be fine